cli/cmd/testdata/inject_emojivoto_pod.golden.yml

```diff
@@ -31,6 +31,10 @@ spec:
       value: 10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
     - name: LINKERD2_PROXY_DESTINATION_PROFILE_NETWORKS
       value: 10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
+    - name: LINKERD2_PROXY_INBOUND_CONNECT_TIMEOUT
```

Comment:
Every golden file got these connect timeouts added, as these where part of the `Values`, and were made for control-plane only and not for user applications?

If thats the case, we can have do have another check in tpl's to make this emitted only for control-plane components :thinking:

Reply:
since these match the defaults in the proxy, this change should be a no-op, right?  But it's more explicit and directly reflects what's in the values.yml.  Seems like a good change to me if that's true.